packages/e2e/local-network/scripts/make-babbage.sh

```diff
@@ -11,6 +11,8 @@ root_path="$(cd "$here/.." && pwd)"
 cd "$root_path"
 
 export PATH=$PWD/bin:$PATH
+export CARDANO_NODE_CHAINDB_LOG_LEVEL="Warning"
```

Comment:
This makes the setting _hard coded_ in the container image, it can't be overridden through environment variables.

```suggestion
export CARDANO_NODE_CHAINDB_LOG_LEVEL=${CARDANO_NODE_CHAINDB_LOG_LEVEL:-Warning}
```

Reply:
Agreed – it was a `warning` before my change, which broke the node, I just changed the case to `Warning` to make it work

Reply:
In that case, the error is not very informative, no stack trace, no anything, just a failed `read`:
```
cardano-node: Prelude.read: no parse
```
Reported [here](https://input-output-rnd.slack.com/archives/CR599HMFX/p1697107953967799).

Reply:
Sorry for my initial comment not specifying this, but the next line is affected by the same problem